extensions/wikia/EditPageLayout/css/core/modules.scss

```diff
@@ -119,6 +132,58 @@
 				width: 75px;
 			}
 		}
+
+		.preview_box{
+			h3 {
+				font-size: 14px;
+				margin: 12px 0 8px;
+			}
+
+			.preview_icon {
+				border: 1px solid $color-page-border;
+				box-sizing: border-box;
+				color: $color-epl-text-secondary;
+				display: block;
+				float: left;
+				font-size: 14px;
+				height: 100px;
+				line-height: 1;
+				text-align: center;
+				width: 128px;
+				&:hover {
```

Comment:
Should be a [whitespace between blocks](https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#emptylinebetweenblocks). Same elsewhere in the changes.